src/org/labkey/test/components/glassLibrary/components/MultiMenu.java

```diff
@@ -226,6 +226,16 @@ public MultiMenuFinder withButtonId(String id)
             return this;
         }
 
+        /**
+         * Looks for the export button using the fa-download class
+         * @return export menu utility
+         */
+        public MultiMenuFinder exportButton()
+        {
+            _locator = Locators.menuContainer().withDescendant(Locator.byClass("fa-download"));
```

Comment:
Make this Locator a bit more precise. It could find a menu that uses the `fa-download` icon in one of the menu items but not for the menu itself. I don't know the exact structure but something like:
```suggestion
            _locator = Locators.menuContainer().withChild(Locator.byClass("fa-download"));
```